src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/GeneralApplicationAfterPaymentCallbackHandler.java

```diff
@@ -57,8 +60,13 @@ private CallbackResponse generalAppAfterPayment(CallbackParams callbackParams) {
             return getCallbackResponse(caseDataBuilder);
         }
 
-        caseDataBuilder.businessProcess(BusinessProcess
-                                            .ready(INITIATE_GENERAL_APPLICATION_AFTER_PAYMENT));
+        if (featureToggleService.isCoSCEnabled()) {
+            caseDataBuilder.businessProcess(BusinessProcess
+                                                .ready(INITIATE_COSC_APPLICATION_AFTER_PAYMENT));
```

Comment:
INITIATE_COSC_APPLICATION_AFTER_PAYMENT
INITIATE_GENERAL_APPLICATION_AFTER_PAYMENT

needs to be in ccd definition